src/auth0-session/handlers/callback.ts

```diff
@@ -62,6 +62,6 @@ export default function callbackHandlerFactory(
     res.writeHead(302, {
       Location: openidState.returnTo || config.baseURL
     });
-    res.end();
+    res.end(openidState.returnTo || config.baseURL);
```

Comment:
Also - I think we should escape these (and the others)

It's probably overkill because modern browsers wont serve this content, but `openidState.returnTo` can contain reflected user input from `/api/auth/login?returnTo=user-input`

I also notice that [express escapes the content in 302](https://github.com/expressjs/express/blob/master/lib/response.js#L935)

There's a util method to escape html here https://github.com/auth0/nextjs-auth0/blob/main/src/utils/errors.ts#L26-L34

FYI @lzychowski